tutorials/tutorial-01.md

```diff
@@ -0,0 +1,117 @@
+# Tutorial 2: Tests for Distributions and samples
+
+For the following exercises and tutorials, you will be building up a codebase in your repository.
+
+During this tutorial, you will copy over some functions from Exercise sheet 1, and modify them to pass the tests that you can find on the bottom of this sheet.
+
+> [!TIP]
+> Reminder on how to run tests in [the setup instructions](https://github.com/RUB-EP1/ExercisesDataAnalysisWS2425/blob/main/exercises/setup.md#back-to-julia-running-tests).
+
+We will need some of the p.d.f.s in un-normalized form (such that the optimizations during fitting can work faster)
+
+1. Gaussian
+2. Polynomial
+3. Relativistic Breit-Wigner
+4. Voigt profile
+
+To pass the tests, their signatures should look like this:
+
+1. `gauss(x; μ, σ, a)`, where `a` is an arbitrary normalization factor
```

Comment:
suggest renaming to make to more specific and clear.
It does not have to be too short